construct/construct.js

```diff
@@ -109,10 +156,36 @@ steal('can/util/string', function (can) {
 		 *     this.last = last;
 		 *   }
 		 * });
+		 * 
+		 * // store a reference to the original newInstance function
+		 * var _newInstance = Person.newInstance;
+		 *
+		 * // override Person's newInstance function
+		 * Person.newInstance = function() {
+		 * // if cache does not exist make it an new object
+		 * this.__cache = this.__cache || {};
+		 * // id is a stingified version of the passed arguments
+		 * var id = JSON.stringify(arguments);
+		 *
+		 * // look in the cache to see if the object already exists
+		 * var cachedInst = this.__cache[id];
+		 * if(cachedInst) {
+		 * 		return cachedInst;
```

Comment:
This creates a warning: https://travis-ci.org/bitovi/canjs/jobs/56051453#L487  I think tabs were mistakingly used right of the `*`.